django/contrib/admin/templates/admin/date_hierarchy.html

```diff
@@ -1,16 +1,14 @@
 {% if show %}
-<div class="xfull">
-<ul class="toplinks">
+<nav class="toplinks">
 {% block date-hierarchy-toplinks %}
 {% block date-hierarchy-back %}
-{% if back %}<li class="date-back"><a href="{{ back.link }}">&lsaquo; {{ back.title }}</a></li>{% endif %}
+{% if back %}<a href="{{ back.link }}" class="date-back">&lsaquo; {{ back.title }}</a>{% endif %}
 {% endblock %}
 {% block date-hierarchy-choices %}
 {% for choice in choices %}
-<li> {% if choice.link %}<a href="{{ choice.link }}">{{ choice.title }}</a>{% else %}{{ choice.title }}{% endif %}</li>
```

Comment:
`li` elements have slightly smaller font i.e. `0.8125rem` instead of `0.875rem` for the `body`. I would add:
```css
.change-list .toplinks a {
    font-size: 0.8125rem;
}
```
to keep the current font size.

Reply:
Ah, thanks, I had no idea. I've added this now.